Oanda/modules/trader/interpreter/interpreter.py

```diff
@@ -17,6 +18,9 @@
 
 allowed_pairs = int_cfg['allowed_pairs']
 
+class MissingRepsonseException(Exception):
```

Comment:
Should be 'Response', not 'Repsonse'

Reply:
Also, Exceptions can be either a Warning or an Error. I would say this is an Error, so we should call it an Error. Also would be nice if it contained its own print text.